Makefile

```diff
@@ -28,11 +28,9 @@ $(EXTENSION)--$(EXTVERSION).sql: setup.sql
 	cat $^ > $@
 
 # Prepare the package for PGXN submission
-DISTVERSION := $(shell git tag -l | tail -n 1 | cut -d 'v' -f 2)
-package: dist dist/$(EXTENSION)-$(DISTVERSION).zip
+DISTVERSION = `git tag -l | tail -n 1 | cut -d 'v' -f 2`
```

Comment:
IIUC The git fatal message is only avoided by the backslashes, but technically it's still executed.

I think it would be cleaner to remove it here and instead add something like that in the package rule (with some extra output to validate the content):

```
package: dist .git
+ 	$(eval DISTVERSION := $(shell git tag -l | tail -n 1 | cut -d 'v' -f 2))
+ 	$(info Generation zip file for version $(DISTVERSION)...)
	git archive --format zip --prefix=$(EXTENSION)-${DISTVERSION}/ --output [...]
```

Reply:
Thanks. I force-pushed the modifications you suggested. Is the commit ok now?